### ISSUE TITLE
Some perses runs fail with "Exception: Programming error: OpenEye atom stereochemistry assumptions failed."

### ISSUE THREAD

**Issue by aizvorski** [choderalab/qmlify#13]
I'm trying to re-run perses using the experiments/mm_data/inputs/run.py script

I noticed that a few of the runs fail, for example:

```
$ python run.py 4
Warning: The aromaticity of the molecule needs to be perceived first.
Traceback (most recent call last):
  File "/home/alex/miniconda3/envs/qmlify-env/bin/perses-relative", line 10, in <module>
    sys.exit(run())
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/perses/app/setup_relative_calculation.py", line 610, in run
    setup_dict = run_setup(setup_options)
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/perses/app/setup_relative_calculation.py", line 356, in run_setup
    trajectory_directory=trajectory_directory, trajectory_prefix=setup_options['trajectory_prefix'])
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/perses/app/relative_setup.py", line 261, in __init__
    small_molecule_forcefield=small_molecule_forcefield, molecules=molecules, cache=small_molecule_parameters_cache)
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/openmmforcefields/generators/system_generators.py", line 206, in __init__
    self.add_molecules(molecules)
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/openmmforcefields/generators/system_generators.py", line 233, in add_molecules
    self.template_generator.add_molecules(molecules)
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/openmmforcefields/generators/template_generators.py", line 124, in add_molecules
    self._molecules.update( { molecule.to_smiles() : molecule for molecule in molecules } )
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/openmmforcefields/generators/template_generators.py", line 124, in <dictcomp>
    self._molecules.update( { molecule.to_smiles() : molecule for molecule in molecules } )
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/openforcefield/topology/molecule.py", line 1841, in to_smiles
    smiles = to_smiles_method(self)
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/openforcefield/utils/toolkits.py", line 1032, in to_smiles
    oemol = OpenEyeToolkitWrapper.to_openeye(molecule)
  File "/home/alex/miniconda3/envs/qmlify-env/lib/python3.7/site-packages/openforcefield/utils/toolkits.py", line 934, in to_openeye
    'Programming error: OpenEye atom stereochemistry assumptions failed.'
Exception: Programming error: OpenEye atom stereochemistry assumptions failed.
Relative calcluation of ligand 6 to 0 complete
```

The failing runs are any that involve ligand 6 or 13.  All others seem to run okay.

Installed versions:
qmlify=g01abd1c (latest from github)
perses=0.5.0=py37_0
openeye-toolkits=2020.1.0=py37_0
openforcefield=0.6.0=py37_1
openforcefields=1.3.0=py_0

How do I fix this?  Is this a version incompatibility or a bug in one of the other packages?

**Comment by dominicrufa** [comment]
@aizvorski , I'm currently in the process of making a new environment for `qmlify`. can you use the latest version of perses [here](https://anaconda.org/omnia/perses)? I vaguely remember this error in `perses` with some other test systems, but can't seem to find the issue right now.

**Comment by j-wags** [comment]
Hi @aizvorski,

Thanks for the detailed environment info. 

I think this is an `openforcefield` <--> `openeye` version mismatch. We had to make a patch release for some underlying changes when OpenEye moved from their 2019 to 2020 toolkits. 

This should be resolved by updating `openforcefield` to a newer version (`0.7.1` or higher). We just released 0.8.1 today, so that would be a good choice :-)  

`conda install -c conda-forge -c omnia "openforcefield >=0.8.1"`


CC: https://github.com/openforcefield/openforcefield/issues/742#issuecomment-706247083

**Comment by aizvorski** [comment]
@dominicrufa @j-wags Thanks guys!  I upgraded and everything works good now

perses=0.8.1=py_0
openeye-toolkits=2020.1.1=py36_0
openforcefield=0.8.0=pyh39e3cac_0
openforcefields=1.3.0=py_0

Really not sure how I ended up with perses=0.5 before, that was not intentional :) 

This really simple environment.yml results in a working environment - it seems perses pulls in anything that's needed; the other dependencies are there just for the analysis notebook.

```
name: qmlify-env
channels:
  - omnia-dev
  - openeye
  - omnia
  - conda-forge
  - defaults
dependencies:
  - perses=0.8.1
  - scikit-learn=0.23.2
  - plotly=4.14.0
  - jupyter
  - pip
  - pip:
    - git+https://github.com/choderalab/qmlify.git@01abd1c
    - git+https://github.com/choderalab/coddiwomple.git@841ac56
    - git+https://github.com/openforcefield/Arsenic.git@0.1.0
```
